packages/compass-import-export/src/components/import-modal.tsx

```diff
@@ -242,7 +222,7 @@ function ImportModal({
       data-testid="import-modal"
       size={fileType === 'csv' ? 'large' : 'small'}
     >
-      <ModalHeader title="Import" subtitle={`To Collection ${ns}`} />
+      <ModalHeader title={`Import to collection ${ns}`} />
```

Comment:
I think we actually want it as it was before, it's the main pattern we follow for modal titles in Compass.

Reply:
OK I reverted it.